### DISCUSSION TITLE
Shelly Plus 1 WEBIF no longer accessible after one minute

### DISCUSSION THREAD

**Discussion by dippes** [arendst/Tasmota#21488] [Q&A] [answered]
I have flashed two brand new Shelly plus 1 with Tasmota 14.0.0. I have installed Tasmota 32-solo1 as firmware and after a minute I can no longer access the WIF.

However, the Shelly is shown as active in the router.

After a restart the WIF is accessible again for about 1 minute.

My other Shellys (no Plus 1) work perfectly.

Does anyone have any idea what this could be?

**Comment by Jason2866** [comment]
To which Router / AP does the device connect?

**Reply by dippes** [reply]
With a Frizbox 6660 Cable
![Screenshot 2024-05-26 081227](https://github.com/arendst/Tasmota/assets/4214724/9353a531-6133-41e5-99fa-284a6abc5d16)
![Screenshot 2024-05-26 081508](https://github.com/arendst/Tasmota/assets/4214724/cccc7932-498c-410d-908a-32d2dda7efb6)


I have also tried tasmota-DE.bin and always the same problem: after about 1 minute the WIF is no longer accessible although it is marked as active in the AP.

**Reply by dippes** [reply]
I would like to flash 13.2.0. Which flashing tool should I use for this? The Tasmotizer 1.2 always displays an error message.
![Screenshot 2024-05-26 083651_Tasmotizer](https://github.com/arendst/Tasmota/assets/4214724/16dcf0b8-4887-4514-b6d7-a6dfb09b9bb9)

**Reply by sfromis** [reply]
If you are flashing the Shelly Plus 1, you are doing it wrong, as this device is based on ESP32. Thus it cannot work flashing a binary for ESP8266 devices.

I'd recommend using the [Tasmota Web Installer](https://tasmota.github.io/install/), which automatically can handle the details around chip type. This also goes if you were trying to flash `tasmota32solo1.bin`, as first flashing of ESP32 devices has to use the `.factory` variant of the binary, which the web installer handles by itself.

**Reply by dippes** [reply]
I've installed the solo1 using the web installer. But how do I get an older firmware version installed?
ht?

**Reply by sfromis** [reply]
You can still use the web installer with drag/drop, after having downloaded the `.factory` binary you want. Current release on https://ota.tasmota.com/tasmota32/release/ and older releases at the bottom of the page. The key is still to make sure that you use the right binary, instead of what you posted about having tried.

**Reply by dippes** [reply]
I have installed 13.3.0. The Wif is now continuously accessible and everything works as it should.
Something is probably not compatible with the Shelly Plus 1 with the current firmware.

**Reply by sfromis** [reply]
I have the similar Shelly Plus 1PM working with the current Tasmota version.

**Reply by dippes** [reply]
Is it perhaps due to the AP?

**Reply by Jason2866** [reply]
Would be not the first issue where the AP is the reason for problems.

**Reply by dippes** [reply]
But it's strange that it works perfectly with 13.3.0. All other Shellys also work without problems.

**Reply by dippes** [reply]
13.4.0 still works fine, but as soon as I use 14.0.0 I have the WIF problem.

What is different?

**Reply by Jason2866** [reply]
The underlying Arduino is changed from 2.0.x to 3.0.0. So every library with precompiled code has changed. Not Tasmota code.

**Reply by Jason2866** [reply]
Tested with my orig. espressif solo1 development board. Works perfectly fine. No issue.

**Reply by arendst** [reply]
My Shelly plus 1PM also keeps responding instantly.

**Reply by dippes** [reply]
How can I find out what the problem is?

**Reply by Jason2866** [reply]
Looks like your AP "thinks" device is disconnected. Search for settings in your Router which could prevent this. Power saving modes for clients. Disable such settings.  Maybe there is a setting to keep a client connections alive.

**Reply by dippes** [reply]
I updated from 13.4.0 to 14.0.0. The update went smoothly.
After about 1 minute the WIF was no longer accessible with the 14.0.0. With the 13.4.0 there are no problems in this regard. That couldn't be the AP if it works with older firmware.

I don't have any energy saving mode active in the AP either. I also deleted the configuration created by the router from the Shelly Plus 1. But nothing helped.

**Reply by arendst** [reply]
Do you have enabled Wemo or Hue emulation enabled? With Hue enabled I do see slow response of GUI too. This is inline with this issue https://github.com/arendst/Tasmota/discussions/21484#discussioncomment-9576897.

It looks indeed we have an issue with UDP on core3 (as used in v14.0.0)

**Reply by dippes** [reply]
Yes, I have activated Belkin Wemo.

**Reply by arendst** [reply]
Could you pls open the GUI and use the console with ``weblog 4``.

Then wait for several minutes for updates to show. After half an hour copy the text from the GUI and paste it here. I'm seeing lots of Multicast disabled and rejoined which shouldn't be there...

**Reply by dippes** [reply]
With Wemo activated?

**Reply by arendst** [reply]
Yes.

**Reply by dippes** [reply]
11:58:08.482 RSL: RESULT = {"WebLog":4}
11:58:13.985 UDP: Packet (119/279)
11:58:20.696 WIF: Prüfe Verbindung…
11:58:40.682 WIF: Prüfe Verbindung…
11:59:00.708 WIF: Prüfe Verbindung…
11:59:20.722 WIF: Prüfe Verbindung…
11:59:40.699 WIF: Prüfe Verbindung…
12:00:00.677 WIF: Prüfe Verbindung…
12:00:20.692 WIF: Prüfe Verbindung…
12:00:40.714 WIF: Prüfe Verbindung…
12:01:00.673 WIF: Prüfe Verbindung…
12:01:20.690 WIF: Prüfe Verbindung…
12:01:40.707 WIF: Prüfe Verbindung…
12:02:00.675 WIF: Prüfe Verbindung…
12:02:20.686 WIF: Prüfe Verbindung…
12:02:40.703 WIF: Prüfe Verbindung…
12:03:00.708 WIF: Prüfe Verbindung…
12:03:04.963 BRY: GC from 24786 to 23782 bytes, objects freed 2/303 (in 1 ms) - slots from 224/380 to 217/380
12:03:04.966 RSL: STATE = {"Time":"2024-05-28T12:03:04","Uptime":"0T00:05:08","UptimeSec":308,"Heap":97,"SleepMode":"Dynamic","Sleep":50,"LoadAvg":19,"MqttCount":0,"Berry":{"HeapUsed":23,"Objects":303},"POWER":"OFF","Wifi":{"AP":1,"SSId":"Router966F24E92D65857CBC","BSSId":"0C:72:74:A2:76:CA","Channel":1,"Mode":"HT20","RSSI":52,"Signal":-74,"LinkCount":1,"Downtime":"0T00:00:04"}}
12:03:05.006 RSL: SENSOR = {"Time":"2024-05-28T12:03:04","Switch1":"OFF","ANALOG":{"Temperature1":7.1},"TempUnit":"C"}
12:03:20.697 WIF: Prüfe Verbindung…
12:03:40.702 WIF: Prüfe Verbindung…
12:03:58.680 CFG: Saved, Anzahl 30, Bytes 4096
12:04:00.698 WIF: Prüfe Verbindung…
12:04:20.719 WIF: Prüfe Verbindung…
12:04:40.679 WIF: Prüfe Verbindung…
12:05:00.697 WIF: Prüfe Verbindung…
12:05:20.719 WIF: Prüfe Verbindung…
12:05:40.682 WIF: Prüfe Verbindung…
12:06:00.700 WIF: Prüfe Verbindung…
12:06:20.709 WIF: Prüfe Verbindung…
12:06:40.669 WIF: Prüfe Verbindung…
12:07:00.684 WIF: Prüfe Verbindung…
12:07:05.945 UPP: Multicast deaktiviert
12:07:06.980 WIF: IPv4 192.168.1.167, mask 255.255.255.0, gateway 192.168.1.1
12:07:07.949 UPP: Multicast (wieder-)verbunden
12:07:08.162 WIF: IPv6 Local fe80::ce7b:5cff:fe86:9558%st1
12:07:10.589 UDP: Packet (119/173)
12:07:10.594 WMO: WeMo Type 2, Antwort gesendet to 192.168.1.137:7897
12:07:11.192 HTP: WeMo-Setup from 192.168.1.137
12:07:20.704 WIF: Prüfe Verbindung…

**Reply by arendst** [reply]
I see you're not using a MQTT broker and also have multicast (re)connects. Thx for the info.

**Reply by dippes** [reply]
I saw that the problem has been fixed. When will it be released?

**Reply by Jason2866** [reply]
In a few minutes it will be here https://ota.tasmota.com/tasmota32/
development build

**Reply by arendst** [reply]
Now! It's in the latest dev binaries.

Final release somewhere in mid-June

**Reply by dippes** [reply]
👍

**Comment by arendst** [comment, marked as answer]
Fixed in v14.0.0.2 See #21470
